docker-compose.override.yaml

```diff
@@ -0,0 +1,5 @@
+version: '3.3'
```

Comment:
shouldn't it be `docker-compose.override.yaml.dist` ?

Reply:
This shouldn't exist. It's on gitignor...